config/functions

```diff
@@ -278,11 +278,22 @@ setup_toolchain() {
     TARGET_LDFLAGS+=" $FLAGS_OPTIM_LTO_OFF"
   fi
 
+  # Linker flags
+  DEFAULT_LINKER="${DEFAULT_LINKER:-bfd}"
+  TARGET_LINKER_FLAGS="-fuse-ld=${DEFAULT_LINKER}"
```

Comment:
You could remove setting this here, and have TARGET_LDFLAGS below set to `TARGET_LDFLAGS+=" ${TARGET_LINKER_FLAGS:--fuse-ld=${DEFAULT_LINKER}}"`

Reason to do this would be that it would only add one 'fuse-ld=xyz' to the output if debugging a package. May be considered less readable here though. Your choice.

Reply:
well this should only add it one more time at `  TARGET_LDFLAGS+=" ${TARGET_LINKER_FLAGS}"` - before only `TARGET_LINKER_FLAGS` is updated/changed 🤔 
```
  # Linker flags
  DEFAULT_LINKER="${DEFAULT_LINKER:-bfd}"
  TARGET_LINKER_FLAGS="-fuse-ld=${DEFAULT_LINKER}"

  # bfd flag
  if flag_enabled "bfd" "no"; then
    TARGET_LINKER_FLAGS=" ${LDFLAGS_OPTIM_BFD}"
  # gold flag
  elif flag_enabled "gold" "no"; then
    TARGET_LINKER_FLAGS=" ${LDFLAGS_OPTIM_GOLD}"
    have_gold="yes"
  # mold flag
  elif flag_enabled "mold" "no"; then
    TARGET_LINKER_FLAGS=" ${LDFLAGS_OPTIM_MOLD}"
  fi
  TARGET_LDFLAGS+=" ${TARGET_LINKER_FLAGS}"
```

Reply:
It's good. I missed it wasn't a '+='.

Reply:
np